test/models/journeyDomain/traderDetails/ConsignmentConsigneeDomainSpec.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.JourneyDomain.traderDetails
+
+import base.SpecBase
+import commonTestUtils.UserAnswersSpecHelper
+import generators.Generators
+import models.domain.{EitherType, UserAnswersReader}
+import models.journeyDomain.traderDetails.ConsignmentConsigneeDomain
+import models.reference.{Country, CountryCode}
+import models.{Address, EoriNumber}
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalacheck.Gen
+import pages.traderDetails.consignment.consignee
+
+class ConsignmentConsigneeDomainSpec extends SpecBase with UserAnswersSpecHelper with Generators {
+
+  "ConsignmentConsigneeDomain" - {
+
+    val consigneeEori    = arbitrary[EoriNumber].sample.value
+    val consigneeName    = Gen.alphaNumStr.sample.value
+    val consigneeAddress = Address("line1", "line2", "postalCode", Country(CountryCode("GB"), "description"))
```

Comment:
Can this be made arbitrary? Looks like we have a `arbitraryTraderAddress` inside `ModelGenerators` (which could probably be renamed to just `arbitraryAddress`)

Reply:
Done